metaflow/metaflow_version.py

```diff
@@ -14,7 +14,7 @@
 __all__ = ("get_version",)
 
 CURRENT_DIRECTORY = path.dirname(path.abspath(__file__))
-INFO_FILE = path.join(CURRENT_DIRECTORY, "INFO")
+INFO_FILE = path.join(path.dirname(CURRENT_DIRECTORY), "INFO")
```

Comment:
So just to confirm, the file named 'INFO' is actually two directories up correct?

Reply:
correct.